tests/resources/modelmesh/sample-minio.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: minio
+spec:
+  ports:
+    - name: minio-client-port
+      port: 9000
+      protocol: TCP
+      targetPort: 9000
+  selector:
+    app: minio
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: minio
+  name: minio
+spec:
+  containers:
+    - args:
+        - server
+        # - /data
+        - /data1
+      env:
+        - name: MINIO_ACCESS_KEY
+          value: AKIAIOSFODNN7EXAMPLE
+        - name: MINIO_SECRET_KEY
+          value: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+      # image: quay.io/cloudservices/minio:latest
```

Comment:
Needs to be removed?

Reply:
Trying to think of a reasonable way to not include it.  Any suggestions?  I suppose I could have the test setup phase generate the key on the fly and use sed on the manifest.  Nothing else really comes to mind.